tldr.py

```diff
@@ -21,10 +21,8 @@
 __client_specification__ = "1.5"
 
 REQUEST_HEADERS = {'User-Agent': 'tldr-python-client'}
-PAGES_SOURCE_LOCATION = os.environ.get(
```

Comment:
Removed as part of the refactor of the Global variable to the `get_pages_source_locations()` function